### PR TITLE
update zaitra logo in meta tags

### PR DESCRIPTION
**Pull request by mjavorka**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -17,10 +17,15 @@
     <meta charset="utf-8">
     <meta content="IE=edge" http-equiv="X-UA-Compatible">
     <meta content="width=device-width, initial-scale=1" name="viewport">
-    <meta content="Custom software solutions in aerospace, machine learning and blockchain areas." name="description">
+    <meta content="SPACE COMPANY focused on Artificial Intelligence and Earth observation" name="description">
```
Comment (rpitonak) [inline _includes/head.html]: Let's keep it `Space company` here.

Comment (rpitonak) [reply _includes/head.html]: also be consistent, Artificial Inteligence Earth Observation

Comment (mjavorka) [reply _includes/head.html]: Fixed

Comment (rpitonak) [reply _includes/head.html]: it is still lower case :D

**Pull request merged**